ambry-named-mysql/src/integration-test/java/com/github/ambry/named/MySqlNamedBlobDbIntegrationTest.java

```diff
@@ -127,10 +139,12 @@ public void testPutGetListDeleteSequence() throws Exception {
 
     // delete the records and check that they cannot be fetched with a get call.
     for (NamedBlobRecord record : records) {
+      time.setCurrentMilliseconds(System.currentTimeMillis());
       DeleteResult deleteResult =
           namedBlobDb.delete(record.getAccountName(), record.getContainerName(), record.getBlobName()).get();
       assertEquals("Unexpected deleted ID", record.getBlobId(), deleteResult.getBlobId());
       assertFalse("Unexpected alreadyDeleted value", deleteResult.isAlreadyDeleted());
+      time.setCurrentMilliseconds(System.currentTimeMillis());
```

Comment:
why do you set timestamp in the same method? Once you set it, it will not change.

Reply:
Bc we need to move the time along. As you mentioned, the time once is set, it will NOT change. That will a problem. Since we are testing the operations per sequence of time. If we don't set here, it's as if we are doing all those operations at the SAME time, which will fail.

Reply:
if you want to make sure that the time moved, you should use `time.sleep` method. otherwise, there is no guarantee that these two `System.currentTimeMillis()` won't return the same result. Those statements can be executed within the same millisecond.